pfcwd/main.py

```diff
@@ -272,15 +276,23 @@ def start_cmd(self, action, restoration_time, ports, detection_time):
         for port in ports:
             if port == "all":
                 for p in all_ports:
+                    pfc_status = self.get_port_pfc_status(p)
+                    if pfc_status == None:
+                        print("SKIPPED: PFC is not enabled on port: {}".format(p))
+                        continue
+
                     self.config_db.mod_entry(
                         CONFIG_DB_PFC_WD_TABLE_NAME, p, None
                     )
                     self.config_db.mod_entry(
                         CONFIG_DB_PFC_WD_TABLE_NAME, p, pfcwd_info
                     )
             else:
-                if port not in all_ports:
+                pfc_status = self.get_port_pfc_status(port)
+                if port not in all_ports or pfc_status == None:
+                    print("SKIPPED: PFC is not enabled on port: {}".format(port))
```

Comment:
In cli common module there should be a function to log to syslog + optionallity to CLI, could you please reuse that?